src/Plugin/Validation/Constraint/StringOrLinkTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\media_entity_twitter\Plugin\Validation\Constraint\StringOrLinkTrait.
+ */
+
+namespace Drupal\media_entity_twitter\Plugin\Validation\Constraint;
+
+use Drupal\Core\Field\FieldItemInterface;
+
+trait StringOrLinkTrait {
```

Comment:
This is actually not StringOrLink but more like WhichField. It will be useful on me_instagram and possibly others too. Should we move to some other place? Media entity?